styles/styles.css

```
@@ -2,12 +2,119 @@
     padding: 0;
     margin: 0;
     box-sizing: border-box;
+    list-style-type: none;
+    list-style: none;
 }
 
-.mainTittle {
-    max-width: 800px;
-    margin: 15px auto;
-    background-color: #ededed;
-    box-shadow: 2px 4px #888888;
+img {
```

Crea una clase para las imágenes y les adaptas el responsive. [Ejemplos](https://www.w3schools.com/howto/howto_css_image_responsive.asp).
```suggestion
.responsive-image {
  max-width: 100%;
  height: auto;
}
```